### ISSUE TITLE
Please re-add setup.py

### ISSUE THREAD

**Issue by hadess** [martinpitt/python-dbusmock#137]
`setup.py` was removed in favour of the newer `pyproject.toml`, unfortunately the only way to install the project using the stock Python install is to use pip, which doesn't have a way to disable network access.

Other build systems exist which might or might not have those features (the search friendly [`build`](https://pypi.org/project/build/), or Poetry), but they're not shipped with a stock Python.

See https://gitlab.gnome.org/GNOME/gnome-bluetooth/-/merge_requests/140

**Comment by martinpitt** [comment]
Right, I just noticed that in PR #131, RPM builds are also still too buggy with pypa build only.

So it seems you are running dbusmock from git master instead of the last release? Interesting, I'll try to be more careful then.
